dist/windows/installer-translations/portugueseBR.nsi

```diff
@@ -1,4 +1,4 @@
-﻿;Installer strings
+;Installer strings
```

Comment:
What is changed here?

Reply:
I would expect to see such a diff pattern for inserting/removing the BOM, but as it turns out the file is still plain UTF-8 and nothing in the beginning of the file up to the start of the second line changed in this commit, so it seems like git just derped here. Everything's fine.